streams/il.m3u

```diff
@@ -17,19 +17,19 @@ https://edge3.uk.kab.tv/live/tv66-heb-high/playlist.m3u8
 https://contact.gostreaming.tv/Knesset/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="Makan33.il" status="error",Makan 33 (1080p)
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/k2slq6Tpsh2bm4wA43P
-#EXTINF:-1 tvg-id="" status="online",Reshet 13 (720p)
+#EXTINF:-1 tvg-id="Channel13.il" status="online",Reshet 13 (720p)
 https://d18b0e6mopany4.cloudfront.net/out/v1/08bc71cf0a0f4712b6b03c732b0e6d25/index.m3u8
-#EXTINF:-1 tvg-id="" status="online",Hidabroot (576p)
+#EXTINF:-1 tvg-id="Hidabroot.il" status="online",Hidabroot (576p)
 https://stream5.shidur.net/htvlive2/smil:live2.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",Hidabroot (576p) [Not 24/7]
+#EXTINF:-1 tvg-id="Hidabroot.il" status="online",Hidabroot (576p) [Not 24/7]
 https://stream71.shidur.net/htvlive2/smil:live2.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",Hidabroot (576p)
+#EXTINF:-1 tvg-id="Hidabroot.il" status="online",Hidabroot (576p)
 https://stream72.shidur.net/htvlive2/smil:live2.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",Musayof (Israel) (240p) [Not 24/7]
 http://wowza.media-line.co.il/Musayof-Live/livestream.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="blocked",As a Child (720p)
+#EXTINF:-1 tvg-id="KanEducational.il" status="blocked",As a Child (720p)
```

Comment:
IS this correct?

Reply:
I wasn't able to run the broadcast itself, so this is just my guess based on the link. But it's really better to leave it blank for now.

Reply:
That link doesn't work for me with a Israel VPN. The new link is

https://www.kankids.org.il/
```
#EXTINF:-1 tvg-id="KanEducational.il" status="online",Kan Educational (720p)
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x7ymw7f
```

Reply:
Found a m3u8 alternative